tests/test_ostree_resolver.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+#coding=utf-8
+# TODO: Remove with in-toto/in-toto#580
+# pylint: disable=bad-indentation
+
+# Copyright New York University and the in-toto contributors
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+<Program Name>
+  test_ostree_resolver.py
+
+<Author>
+  Aditya Sirish A Yelgundhalli <aditya.sirish@nyu.edu>
+
+<Started>
+  May 04, 2023
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Test ostree resolver.
+
+"""
+
+import os
+import unittest
+
+from in_toto.exceptions import OSTreeRefNotFound
+from in_toto.resolver import OSTreeResolver
+
+class TestOSTreeResolver(unittest.TestCase):
+    """Test OSTree resolver implementation. """
+
+    def test_hash_for_branch_with_base_path(self):
+        resolver = OSTreeResolver(base_path="tests/resolver/ostree_repo")
```

Comment:
Using paths relative to cwd makes the test suite less flexible. I usually use paths relative to the test module, e.g.:
https://github.com/in-toto/in-toto/blob/fbbe7927257086e832deb8a860392000eb58f489/tests/test_in_toto_match_products.py#L15-L17